stable/admin/templates/rolebinding.yaml

```diff
@@ -11,14 +11,16 @@ subjects:
 - kind: ServiceAccount
   name: {{ default "nuodb" .Values.nuodb.serviceAccount }}
 {{- if eq (include "defaulttrue" .Values.nuodb.addClusterRoleBinding) "true" }}
+{{- $namespace := default .Release.Namespace .Values.admin.namespace | trunc 50 | trimSuffix "-" -}}
+{{- $adminName := include "admin.fullname" . }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: nuodb-kube-inspector
+  name: {{ $namespace }}-{{ $adminName }}-nuodb-kube-inspector
```

Comment:
This will result in something like `nuodb-admin-nuodb-cluster0-nuodb-kube-inspector`. A lot of repeating `nuodb`. Does it make sense to define another template for cluster-scoped resources? For example:

```go
{{- define "admin.fullclustername" -}}
{{- $name := include "admin.fullname" . -}}
{{- $ns := default .Release.Namespace .Values.admin.namespace | trunc 50 | trimSuffix "-" -}}
{{- if contains $name $ns -}}
{{- printf "%s" $name -}}
{{- else -}}
{{- printf "%s-%s" $name $ns -}}
{{- end -}}
{{- end -}}
```

and then use it as `{{ include "admin.fullclustername" . }}-kube-inspector`.

Reply:
Good idea